### PR TITLE
Feature tvdata with posters

### PR DESCRIPTION
**Pull request by mujtabahasni**
Adding posters to distinguish similarity named tv series and make search results easier to identify.  Summaries are also being included in the data, but not shown yet in the view.

### CODE REVIEW

File: src/app/services/tvdb.service.ts [hunk 1]
```diff
@@ -10,11 +10,21 @@ const API_BASE_URL = 'http://api.tvmaze.com';
...
 }
 
+function getPlaceholderUrl(title: string) {
+  return `https://via.placeholder.com/210x295?text=${encodeURI(title)}`;
+}
+
 const itemToShowData = (item: TvDbResponseItem): TvShowData => ({
   title: item.show.name,
+  summary: item.show.summary,
+  posterUrl: item.show.image ? item.show.image.medium : getPlaceholderUrl(item.show.name),
```
Comment (andrewthauer) [inline src/app/services/tvdb.service.ts]: If you wanted you could use `ramda`'s `ifElse` function for this.

File: src/app/tvshows/tvshows-search.component.ts [hunk 2]
```diff
@@ -17,9 +17,7 @@ import { TvShowData } from './store/tvshows.models';
   <div *ngIf="selectors.isFetching$ | async">
     <progress value="80" max="100">fetching...</progress>
   </div>
-  <ul>
-    <li *ngFor="let show of (selectors.tvshows$ | async)">{{ show.title }}</li>
-  </ul>
+  <app-tvshow-poster *ngFor="let show of (selectors.tvshows$ | async)" [show]="show"></app-tvshow-poster>
```
Comment (andrewthauer) [inline src/app/tvshows/tvshows-search.component.ts]: You might want to consider splitting the search and `results` into separate components and using a container to tie them together.

File: src/app/tvshows/tvshows-tvdata.component.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { Component, Input } from '@angular/core';
+import { TvShowData } from './store/tvshows.models';
+
+@Component( {
+  selector: 'app-tvshow-poster',
+  template: `
+  <table border="1" style="display:inline-block">
```
Comment (andrewthauer) [inline src/app/tvshows/tvshows-tvdata.component.ts]: How come your using a table for this? I don't see why it's necessary to use a able. This is also an anti-pattern to use table for layout only.

**Review by andrewthauer — CHANGES_REQUESTED** [review]
Overall seems fine to me. Left a couple of comments.

File: src/app/services/tvdb.service.ts (hunk 1 above)
Comment (mujtabahasni) [reply src/app/services/tvdb.service.ts]: Will do.  I want to use Ramda wherever I can.

File: src/app/tvshows/tvshows-search.component.ts (hunk 2 above)
Comment (mujtabahasni) [reply src/app/tvshows/tvshows-search.component.ts]: Good idea. Will do.

File: src/app/tvshows/tvshows-tvdata.component.ts (hunk 3 above)
Comment (mujtabahasni) [reply src/app/tvshows/tvshows-tvdata.component.ts]: This is temporary, I do plan to replace this with angular-material cards.    Using tables is lazy of me though...  I can fix this quickly.

**Pull request merged**